scripts/workers/submission_worker.py

```diff
@@ -167,7 +167,14 @@ def create_dir(directory):
         Creates a directory if it does not exists
     """
     if not os.path.exists(directory):
-        os.makedirs(directory)
+        try:
```

Comment:
Can we revert this, as this is not part of the task. You can submit a separate Pr for this

Reply:
Of course! After I revert this, I will be using `assert os.path.exists()` to detect the failure of file creation, since I think we can't detect it with mock_logger without being logged.